packages/aragon-api/src/index.js

```diff
@@ -214,10 +251,16 @@ export class AppProxy {
    *
    * @param  {Function} reducer A function that reduces events to a state. This can return a Promise that resolves to a new state.
    * @param  {Observable[]} [events] An optional array of `Observable`s to merge in with the internal events observable
+   * @param  {Function} [init] An   optional array of `Observable`s to merge in with the internal events observable
    * @return {Observable} An [RxJS observable](http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html) that emits the application state every time it changes. The type of the emitted values is application specific.
    */
-  store (reducer, events = [empty()]) {
-    const initialState = this.state().pipe(first())
+  store (reducer, events = [empty()], init) {
```

Comment:
I like that this is kind of backwards compatible, but I think it'd be much more clear if it were something like:

```js
store (init, reducer, { externalEvents })
```

Reply:
That could work